enacts/onset/controls.py

```diff
@@ -0,0 +1 @@
+../../controls.py
```

Comment:
When I set up the multi-page configuration initially, I was just trying to get something out as quickly as possible given the time constraints, but I'm not really sure if I did things the absolute best way. I wonder if there is any way we can put controls.py somewhere such that all the individual sub-apps can access it without having to symlink it.

Reply:
This will be taken care of by packaging pingrid (working name) as a library.